### DISCUSSION TITLE
Prestashop Plugin

### DISCUSSION THREAD

**Discussion by rieders** [inventree/InvenTree#5640] [Ideas]
Hello.
I'm still relatively new to Inventree.
Is there a way to connect an online shop with Inventree in real time.
Can I add certain stock items to the online shop?
As soon as it is sold will it be removed from inventory?

I'm specifically interested in Prestashop.

Greetings

**Comment by wolflu05** [comment]
Yes, that is definitely possible with a plugin. There is already one for a [Shopify integration](https://github.com/matmair/inventree-shopify). To get started with plugin development, you could take a look at our [develop a plugin docs](https://docs.inventree.org/en/stable/extend/how_to_plugin/), the [django getting started docs](https://docs.djangoproject.com/en/4.2/intro/tutorial01/) (recommended to read if you want to integrate deeper into inventree, e.g. by using the [app mixin](https://docs.inventree.org/en/latest/extend/plugins/app/)) and also at [existing plugins](https://inventree.org/plugins). For a plugin development setup, I can recommend using [devcontainers](https://docs.inventree.org/en/latest/develop/devcontainer/#devcontainer), we have a dedicated section for how to [set them up for plugin development](https://docs.inventree.org/en/latest/develop/devcontainer/#plugin-development) too. That all should give you a good starting point for plugin development. If you have any further questions, feel free to ask.

**Comment by SergeoLacruz** [comment]
The suoolier-panel plugin might be a goot starting point 
https://github.com/sergeolacruz/inventree-supplier-panel
This actually works only with Mouser. I tried to add Digikey but their auth system is more complex.
